settings/DevHome.Settings/DevHome.Settings.csproj

```diff
@@ -38,6 +38,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\common\DevHome.Common.csproj" />
+    <ProjectReference Include="..\..\tools\QuietBackgroundProcesses\DevHome.QuietBackgroundProcesses.ElevatedServer.Projection\DevHome.QuietBackgroundProcesses.ElevatedServer.Projection.csproj" />

```

Comment:
Please add new reference to project diagram here https://github.com/microsoft/devhome/blob/main/docs/architecture.md